keras_cv/layers/preprocessing/random_resized_crop.py

```diff
@@ -0,0 +1,91 @@
+from keras import backend
+from keras.engine import base_layer
+from keras.engine import base_preprocessing_layer
+from keras.layers.preprocessing import preprocessing_utils as utils
+from keras.utils import image_utils
+from keras.utils import tf_utils
+import numpy as np
+import tensorflow as tf
+import random
+from tensorflow.python.util.tf_export import keras_export
+from tensorflow.tools.docs import doc_controls
+
+H_AXIS = -3
+W_AXIS = -2
+
+
+class RandomResizedCrop(base_layer.Layer):
```

Comment:
lets subclass tf.keras.__internal__.layers.BaseImageAugmentationLayer instead of base_layer.Layer.

Check out
https://github.com/keras-team/keras-cv/blob/master/keras_cv/layers/preprocessing/random_shear.py

Reply:
Done : Added a subclass tf.keras.internal.layers.BaseImageAugmentation.Layer  in place of base_layer.Layer